node/src/components/linear_chain.rs

```diff
@@ -594,3 +601,34 @@ where
         }
     }
 }
+
+#[derive(Debug)]
+struct LinearChainMetrics {
+    /// Time in milliseconds since the unix epoch that the last block was added.
+    time_of_last_added_block: IntGauge,
```

Comment:
Where is this metric updated? Also, we already have very similar metrics in `BlockExecutor` and `Consensus` so why do we need another one?

Reply:
I had updated the description but I neglected to push the last commit.